packages/react/src/PageMenu/PageMenu.tsx

```diff
@@ -0,0 +1,72 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import React, {
+  AnchorHTMLAttributes,
+  ButtonHTMLAttributes,
+  ForwardedRef,
+  forwardRef,
+  HTMLAttributes,
+  PropsWithChildren,
+} from 'react'
+import { Icon } from '../Icon'
+
+interface PageMenuComponent
+  extends React.ForwardRefExoticComponent<
```

Comment:
You can import these types directly, like so:

```suggestion
  extends ForwardRefExoticComponent<
```